src/views/AddItem.jsx

```diff
@@ -1,9 +1,17 @@
 import { useState } from 'react';
 import { addItem } from '../api/firebase';
 
+// sets number of days for each future purchase estimate variable
+const numDaysInEstimate = {
+	soon: '7',
+	kindOfSoon: '14',
+	notSoon: '30',
+};
+
+// sets initial default values in form fields and deconstruct form field variables
+const initialState = { itemName: '', estimate: '7' };
```

Comment:
```suggestion
const initialState = { itemName: '', estimate: numDaysInEstimate.soon };
```

So the beauty about the constant is that all references to this number should use the constant, so it makes it clear what the number is referring to.